packages/calcite-components/src/components/navigation-logo/navigation-logo.scss

```diff
@@ -42,39 +59,34 @@
   @apply ps-0;
 }
 
-:host(:active) .anchor {
-  @apply text-color-1;
-}
-
 :host([active]) .anchor {
-  @apply text-color-1;
-  border-color: var(--calcite-color-brand);
-  --calcite-icon-color: var(--calcite-color-brand);
+  --calcite-icon-color: var(--calcite-navigation-logo-icon-color-press, var(--calcite-color-brand));
```

Comment:
this should be 
```css
.icon {
color: var(--calcite-navigation-logo-icon-color-press, var(--calcite-icon-color, var(--calcite-color-brand)));
}
```

Reply:
does the state tokens apply here? 

we can use one token 

```
.icon {
  color: var(--calcite-navigation-logo-text-color, var(--calcite-internal-navigation-logo-text-color, var(--calcite-icon-color, inherit)));
}

:host([active]),
:host(:active) {
  .icon {
 --calcite-internal-navigation-logo-text-color: var(--calcite-icon-color, var(--calcite-color-brand)));
  }
}